README.md

```diff
@@ -57,6 +58,15 @@ Paid support is available for \$135 USD per hour<sup>[1][footnote-1]</sup>.
 [prosperity public license]: https://prosperitylicense.com/
 [issue-tracker]: https://github.com/zinc-collective/convene/issues
 
+## Architecture
+
+At present, Convene is split into three modules:
+
+- `infrastructure`, which contains infrastructure management code for Packer,
+  Terraform and Ansible.
+- `web` which provides the human interface for Convene.
+- `api` which stores data long-term and provides a programmatic interface.
```

Comment:
@cjerozal / @user512 - Does this make sense from a "how do we want to split out responsibilities" for now? I feel like the majority of the work will be in the `infrastructure` module; but I also expect that as we start building out WebRTC UI bits that we will want those to live in something like `web` so folks don't need to know too much about Packer and such.

`api` may be extraneous; as we could most likely bundle that in with web.

Reply:
I think this makes sense.